### PR TITLE
participants route

### PR DESCRIPTION
**Pull request by SURAJSHARMA-ind**
Add get participants route

### CODE REVIEW

File: controller/participants.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+const getparticipants = async (req, res) => {
+  try {
+    const participants = await participantsModel.find();
+
+    if (participants.length === 0) {
+      return res.send({
+        success: true,
+        status: 200,
+        message: "No participants found",
+        data: [],
```
Comment (JeetuGuptaa) [inline controller/participants.js:12]: data : {
participants : []
}

File: controller/participants.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      });
+    }
+
+    return res.send({
+      success: true,
+      status: 200,
+      message: "Participants retrieved successfully",
+      data: participants,
```
Comment (JeetuGuptaa) [inline controller/participants.js:20]: data : {
participants 
}

File: controller/participants.js (hunk 1 above)
Comment (SURAJSHARMA-ind) [reply controller/participants.js]: done.

File: controller/participants.js (hunk 2 above)
Comment (SURAJSHARMA-ind) [reply controller/participants.js]: done

**Review by JeetuGuptaa — APPROVED** [review]

**Pull request merged**